pyaerocom/aeroval/collections.py

```diff
@@ -110,9 +111,24 @@ def web_interface_names(self) -> list:
         """
         return self.keylist()
 
-    def to_json(self) -> str:
-        """Serialize ModelCollection to a JSON string."""
-        return json.dumps({k: v.dict() for k, v in self._entries.items()}, default=str)
+    def json_repr(self) -> dict:
```

Comment:
I would call this function `as_dict` or `to_dict`. That dicts are easier to use with the json-module is only implicit.